configure

```diff
@@ -2122,8 +2122,8 @@ printf "%s\n" "unknown" >&6; }
             "$sys_version_minor" -lt "$min_version_minor"; then
             { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: pending-end-of-support" >&5
 printf "%s\n" "pending-end-of-support" >&6; }
-            { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: On $end_support_date, the AWS CLI v2 will drop support for macOS versions below $min_version. macOS $ac_macos_version was detected for this system. Please upgrade to macOS version $min_version or later to ensure compatibility with future versions of AWS CLI v2. For more information, please visit: https://aws.amazon.com/blogs/developer/macos-support-policy-updates-for-the-aws-cli-v2/" >&5
-printf "%s\n" "$as_me: WARNING: On $end_support_date, the AWS CLI v2 will drop support for macOS versions below $min_version. macOS $ac_macos_version was detected for this system. Please upgrade to macOS version $min_version or later to ensure compatibility with future versions of AWS CLI v2. For more information, please visit: https://aws.amazon.com/blogs/developer/macos-support-policy-updates-for-the-aws-cli-v2/" >&2;}
+            { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: On $end_support_date, the AWS CLI v2 dropped support for macOS versions below $min_version. macOS $ac_macos_version was detected for this system. Please upgrade to macOS version $min_version or later to ensure compatibility with future versions of AWS CLI v2. For more information, please visit: https://aws.amazon.com/blogs/developer/macos-support-policy-updates-for-the-aws-cli-v2/" >&5
+printf "%s\n" "$as_me: WARNING: On $end_support_date, the AWS CLI v2 dropped support for macOS versions below $min_version. macOS $ac_macos_version was detected for this system. Please upgrade to macOS version $min_version or later to ensure compatibility with future versions of AWS CLI v2. For more information, please visit: https://aws.amazon.com/blogs/developer/macos-support-policy-updates-for-the-aws-cli-v2/" >&2;}
```

Comment:
We should probably change the wording in all the places we have this to indicate this is no longer forward looking but a requirement to get updates starting tomorrow.

```suggestion
printf "%s\n" "$as_me: WARNING: On $end_support_date, the AWS CLI v2 dropped support for macOS versions below $min_version. macOS $ac_macos_version was detected for this system. Please upgrade to macOS version $min_version or later to use up-to-date versions of AWS CLI v2. For more information, please visit: https://aws.amazon.com/blogs/developer/macos-support-policy-updates-for-the-aws-cli-v2/" >&2;}
```